### ISSUE TITLE
[ui] show keyboard shortcut on hover everywhere

### ISSUE THREAD

**Issue by oliursahin** [marchhq/march#141]

**Comment by oliursahin** [comment]
tesitng webhook

**Comment by Ameerjafar** [comment]
@oliursahin can you assign this issue to work on.

**Comment by oliursahin** [comment]
@Ameerjafar this issue is blocked by another issue: https://github.com/marchhq/march/issues/84

Please pick something else to work on for now
